ksql-engine/src/test/java/io/confluent/ksql/util/SchemaUtilTest.java

```diff
@@ -16,31 +16,324 @@
 
 package io.confluent.ksql.util;
 
+
+import org.apache.kafka.connect.data.Field;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaBuilder;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.sameInstance;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Optional;
+
 public class SchemaUtilTest {
 
+  Schema schema;
+
+  @Before
+  public void init() {
+    schema = SchemaBuilder.struct()
+        .field("ordertime".toUpperCase(), org.apache.kafka.connect.data.Schema.INT64_SCHEMA)
+        .field("orderid".toUpperCase(), org.apache.kafka.connect.data.Schema.INT64_SCHEMA)
+        .field("itemid".toUpperCase(), org.apache.kafka.connect.data.Schema.STRING_SCHEMA)
+        .field("orderunits".toUpperCase(), org.apache.kafka.connect.data.Schema.FLOAT64_SCHEMA)
+        .field("arraycol".toUpperCase(), SchemaBuilder.array(org.apache.kafka.connect.data.Schema.FLOAT64_SCHEMA))
+        .field("mapcol".toUpperCase(), SchemaBuilder.map(org.apache.kafka.connect.data.Schema.STRING_SCHEMA, org.apache.kafka.connect.data.Schema.FLOAT64_SCHEMA))
+        .build();
+  }
+
   @Test
-  public void testGetJavaType() {
-    Class booleanClazz = SchemaUtil.getJavaType(Schema.BOOLEAN_SCHEMA);
-    Class intClazz = SchemaUtil.getJavaType(Schema.INT32_SCHEMA);
-    Class longClazz = SchemaUtil.getJavaType(Schema.INT64_SCHEMA);
-    Class doubleClazz = SchemaUtil.getJavaType(Schema.FLOAT64_SCHEMA);
-    Class StringClazz = SchemaUtil.getJavaType(Schema.STRING_SCHEMA);
-    Class arrayClazz = SchemaUtil.getJavaType(SchemaBuilder.array(Schema.FLOAT64_SCHEMA));
-    Class mapClazz = SchemaUtil.getJavaType(SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.FLOAT64_SCHEMA));
+  public void shouldCreateCorrectAvroSchemaWithNullableFields() {
```

Comment:
why do we have one `SchemaUntil` class, but two test classes?  And why does it look like your PR swaps the contents of these two files?

Reply:
There's still 2 - this one and the one in common.